src/externals/pio2/src/clib/pio_c_template.c

```diff
@@ -0,0 +1,54 @@
+int PIO_function()
```

Comment:
Same here, pio_c_template.c was also deleted (Sep 17, 2016) from pio2 master (https://github.com/PARALLELIO/ParallelIO.git) and is no longer needed